### ISSUE TITLE
CSS doesn't work on <ul> after page break

### ISSUE THREAD

**Issue by Whip** [dompdf/dompdf#1998]
I have some css design applied to make a `<ul>` tag look interesting. Problem is it works fine on the first page but after that, it pretty much ignores all the css. Please take a look at the numbers in red in the screenshot below. Other elements that follow this, work fine that are starting on 2nd page and continuing on 3rd page.

Here's the code:
```
<ul class="itinerary">
	<li>
		<span class="number">1</span>
		<h3>Day 1: Arrive Delhi</h3>
		<p>On arrival at Delhi airport or railway station, our representative will meet you to pick you up and transfer to your hotel. Rest of the day is at your leisure. Stay overnight at the hotel in Delhi.</p>
	</li>
	<li>
		<span class="number">2</span>
		<h3>Day 2: City Tour of Delhi</h3>
		<p>This morning after breakfast first you will visit old Delhi. The Raj Ghat, Red fort &amp; Jama Masjid are important places to visit, you may also visit Chandni Chowk, spice market etc. Later on proceed to visit the New Delhi. The Qutub Minar, tallest stone tower in India, imposing India Gate and the Rastrapathi Bhawan, Laxmi Narayan Temple, Lotus Temple and, newly built Akshardham temple are of much significance. By evening return to your hotel for overnight stay.</p>
	</li>
	...
</ul>
```
CSS
```
.itinerary{
	padding: 0;
  list-style: none;
	position: relative;
}
.itinerary:before {
	content: '';
	width: 2px;
	height: 100%;
	border-left: 2px dashed #d14334;
	position: absolute;
	left: 19px;
}
.itinerary li{
	position: relative;
	padding-left: 70px;
	margin-bottom: 70px;
}
.itinerary li .number{
	position: absolute;
	width: 40px;
	height: 40px;
	line-height: 36px;
	text-align: center;
	background: #d14334;
	color: #FFF;
	font-size: 1.7rem;
	top: 0;
	left: 0;
}
```
![138](https://user-images.githubusercontent.com/11595460/66189793-346d8e00-e6a8-11e9-8547-464888017476.jpg)

**Comment by bsweeney** [comment]
Guessing this is probably related to the style property reset on page break, but we'll have to look into it a bit more before we can say for sure.

**Comment by GoldraK** [comment]
I have the same problem. :(

**Comment by Mellthas** [comment]
There are two issues here:

* The fixed `display` style (`inline` → `block`) of the absolutely positioned `.number` elements is lost after a page break. This is the same issue as #2271 and addressed by the second commit in #2576. This can be worked around with the current version by explicitly setting `display: block` for the absolutely positioned `span` elements.
* The dashed border is only displayed on the first page because absolutely positioned elements (the `::before` content here) are not paged. This is related to #1138. The case in the sample here can be easily addressed by specifying the dashed border on the `.itinerary` element itself (and adapting offsets/margins/padding on the other elements accordingly).
